CS483/CS483/Kartaclysm/Components/ComponentSimplePhysics.h

```diff
@@ -39,6 +39,7 @@ namespace Kartaclysm
 		virtual void Init() override;
 		virtual void Update(const float p_fDelta) override;
 
+		void SetTrackHeight(float p_fTrackHeight) { m_fGroundHeight = p_fTrackHeight; /*m_fPreviousHeight = p_fTrackHeight;*/}
```

Comment:
If the commented code isn't working, it shouldn't be left here without a TODO or something saying why it isn't just deleted.